sql/moz-fx-data-shared-prod/accounts_db_external/fxa_recovery_phones_v1/query.sql

```diff
@@ -0,0 +1,20 @@
+SELECT
+  TO_HEX(uid) AS uid,
+  phoneNumber,
+  SAFE.TIMESTAMP_MILLIS(SAFE_CAST(createdAt AS INT)) AS createdAt,
+  SAFE.TIMESTAMP_MILLIS(SAFE_CAST(lastConfirmed AS INT)) AS lastConfirmed,
+  lookupData
```

Comment:
This is a JSON column.  Does it need anything special here?

Reply:
Oddly the [federated query MySQL to BigQuery type mapping](https://cloud.google.com/bigquery/docs/reference/standard-sql/federated_query_functions#mysql_mapping) docs don't mention the JSON type, but based on a couple other tables with JSON columns already being synced it appears such columns come across in BigQuery as strings rather than [BigQuery's native JSON type](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type).

So if you do want it stored as BigQuery's JSON type you'll need to convert it with [`PARSE_JSON()`](https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#parse_json):
```suggestion
  PARSE_JSON(lookupData) AS lookupData
```

Try running that manually first for both the prod and stage tables though, because it's possible `PARSE_JSON()` might error for some of those values (e.g. if any come across as just an empty string it won't like that).  If errors do occur, you could either not do the JSON conversion, or call it like `SAFE.PARSE_JSON()` which will silently return nulls when it encounters errors.

Reply:
I added the SAFE prefix.  Might as well.  I can't test it until the permissions are fixed.